doc/users/whats_new.rst

```diff
@@ -453,65 +479,141 @@ figure where the subplots will be created. If ``fig`` is ``None`` (default)
 then the method will use the current figure retrieved by
 :func:`~matplotlib.pyplot.gcf`.
 
-Example
-~~~~~~~
+
```

Comment:
this entry could be merged with the addition of figure.subplots?

Reply:
eh, I like the sorting where they are.